package.json

```diff
@@ -1,23 +1,22 @@
 {
-  "name": "peerflix",
+  "name": "@meteor314/stream",
   "description": "Streaming torrent client for Node.js",
-  "version": "0.39.0",
-  "author": "Mathias Buus (@mafintosh)",
+  "version": "1.0.5",
+  "author": "@meteor314",
   "license": "MIT",
-  "homepage": "https://github.com/mafintosh/peerflix",
+  "homepage": "https://github.com/meteor314/stream",
   "main": "index.js",
   "bugs": {
-    "url": "https://github.com/mafintosh/peerflix/issues"
+    "url": "https://github.com/meteor314/stream/issues"
   },
   "repository": {
     "type": "git",
-    "url": "git://github.com/mafintosh/peerflix.git"
+    "url": "git://github.com/meteor314/stream.git"
   },
   "bin": {
-    "peerflix": "./app.js"
+    "stream": "app.js"
   },
```

Comment:
This PR is fully appropriating the package, for a very small feature add. It smells like a hacking attempt.